test/integration/repos_test.go

```diff
@@ -141,7 +138,7 @@ func TestRepositories_EditBranches(t *testing.T) {
 			DismissStaleReviews: true,
 		},
 		EnforceAdmins: &github.AdminEnforcement{
-			URL:     &enforcementURL,
+			URL:     github.String(enforcementURL),
```

Comment:
The reason I suggested `github.String` is so that you could inline `enforcementURL `:

```Go
URL: github.String("https://api.github.com/repos/" + *repo.Owner.Login + "/" + *repo.Name + "/branches/master/protection/enforce_admins"),
```